types/ember-data/test/belongs-to.ts

```diff
@@ -7,11 +7,17 @@ class Folder extends DS.Model {
     name = DS.attr('string');
     children = DS.hasMany('folder', { inverse: 'parent' });
     parent = DS.belongsTo('folder', { inverse: 'children' });
+
+    @DS.belongsTo('owner') owner!: Owner;
+}
+
+class Owner extends DS.Model {
 }
 
 declare module 'ember-data/types/registries/model' {
     export default interface ModelRegistry {
         folder: Folder;
+        owner: Owner;
```

Comment:
Could we name this something else? I had to hunt around to make sure that this wasn't the `owner` from RFC-232 tests (i.e., `this.owner` (ApplicationInstance) )

Reply:
Sure. I've renamed it.